consensus/src/marshal/ingress/mailbox.rs

```diff
@@ -71,14 +100,31 @@ impl<V: Variant, B: Block> Mailbox<V, B> {
         Self { sender }
     }
 
-    /// Get is a best-effort attempt to retrieve a given block from local
+    /// A request to retrieve the information about the highest finalized block.
+    pub async fn get_latest(&mut self) -> oneshot::Receiver<Option<(u64, B::Commitment)>> {
```

Comment:
It is odd that this exists if `get_block(Identifier::Latest)` also exists.

From a user experience point of view, I actually prefer having a trio of `get_latest`, `get_by_height`, `get_by_commitment` (or `get_by_digest`), so I don't have to manually construct an `Identifier::Height(height)`.